sync-orders-to-bigQuery/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "moltin-google-bigquery",
```

Comment:
We need to remove everything here except dev/dependencies and add `"private": true`.

See https://github.com/moltin/integration-examples/blob/master/sync-catalog-to-algolia/package.json for example. This prevents it from accidentally being pushed to npm.

Reply:
Done